src/components/PostDetails/PostDetails.jsx

```diff
@@ -1,45 +1,124 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+
+import PropTypes from 'prop-types';
+
 import { NewCommentForm } from '../NewCommentForm';
+import { Loader } from '../Loader';
+
 import './PostDetails.scss';
 
```

Comment:
```suggestion
import React, { useState, useEffect } from 'react';
import PropTypes from 'prop-types';
import { NewCommentForm } from '../NewCommentForm';
import { Loader } from '../Loader';
import './PostDetails.scss';

```
No need in this empty lines